dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcCpExtension.java

```diff
@@ -15,17 +15,20 @@
  */
 package io.helidon.dbclient.jdbc;
 
-import com.zaxxer.hikari.HikariConfig;
+import java.util.function.Consumer;
 
 /**
  * Interceptor to handle connection pool configuration.
+ * Interceptor does not have access to connection pool implementation so connection pool must provide
+ * consumers for provided configuration values.
  */
-public interface HikariCpExtension {
+public interface JdbcCpExtension {
```

Comment:
This is not a good name. Please rename to `JdbcConnectionPoolExtension`. Do not abbrevieate in Helidon APIs (except for very specific and well known cases, such as Http)

Reply:
Moved to hikari module and renamed to HikariMetricsRegistry.